src/Dotnet.Script.Core/ScriptCompiler.cs

```diff
@@ -34,6 +34,12 @@ static ScriptCompiler()
             {
                 CSharpScript.Create<object>("1", ScriptOptions.Default, typeof(CommandLineScriptGlobals), new InteractiveAssemblyLoader()).RunAsync(new CommandLineScriptGlobals(Console.Out, CSharpObjectFormatter.Instance)).GetAwaiter().GetResult();
             });
+
+            // reset default scripting mode to latest language version to enable C# 8.0 features
+            // this is not needed once Roslyn 3.0 stable ships
+            var csharpScriptCompilerType = typeof(CSharpScript).GetTypeInfo().Assembly.GetType("Microsoft.CodeAnalysis.CSharp.Scripting.CSharpScriptCompiler");
+            var parseOptionsField = csharpScriptCompilerType?.GetField("s_defaultOptions", BindingFlags.Static | BindingFlags.NonPublic);
+            parseOptionsField?.SetValue(null, new CSharpParseOptions(LanguageVersion.CSharp8, kind: SourceCodeKind.Script));
```

Comment:
Does this execute before 
https://github.com/filipw/dotnet-script/blob/cf01f703499220b3bc48ef329d3458dccb4af40f/src/Dotnet.Script.Core/Interactive/InteractiveRunner.cs#L30

Maybe we should add a nullable type test for the REPL too?

Reply:
there is no way to make nullable work yet because it only emits and we swallow warnings at the moment. we also don't expose the ability to treat warnings as errors. I will open a separate issue to handle that